modules/update-preparing-evaluate-apirequestcount.adoc

```diff
@@ -32,6 +32,14 @@ installplans.v1alpha1.operators.coreos.com                     93
 ...
 ----
 +
+[IMPORTANT]
+====
+You can safely ignore the following entries that appear in the results:
+
+* `system:serviceaccount:kube-system:generic-garbage-collector` appears in the results because it walks through all registered APIs searching for resources to remove.
+* `system:kube-controller-manager` appears in the results because, for quota purposes, it walks through all resources to ensure that they are accounted for.
```

Comment:
```suggestion
* `system:kube-controller-manager` appears in the results because it walks through all resources to ensure that they are accounted for.
```

Just to maintain a little bit of parallelism with what is above. :) Not required though.

Reply:
Thanks yeah I know this was a bit awkward. I would have wanted to put it at the end, but really could since the end already ended with a "for" :smile: 

@soltysh Do you think it's important to mention about the quota here? Or perhaps suggest another way to work it in, if so?

Reply:
Maybe this?

> ...because it walks through all resources to count them while enforcing quotas.

Reply:
That sounds good to me if @soltysh agrees. I updated the PR per this suggetion

Reply:
I like Trevor's suggestion :+1: I'd like to be clear what exactly is causing that to show up there. KCM is big enough that this extra bit of details is important.